Colombia/transform_load_raw_dlt.py

```diff
@@ -513,7 +513,9 @@ def col_central_boost_silver_from_raw():
         ) & ~col("pension")), "Goods and services"
       ).when(
         (col("subsidy") & ~col("pension")), "Subsidies"
-      ).otherwise( # Colombia has no "Other grants and transfers"
+      ).when(
+        lower(col('econ2')).startswith("servicio") & ~col('econ1').contains('Adquisición de Activos Financieros'), 'Interest on debt'
```

Comment:
@yukinko-iwasaki if I recall corerctly 'Adquisición de Activos Financieros' should have been exlcuded already somewhere above as that condition appears for many economic & functional category tagging so it has been generalized. Can you check if it's the case and the `~col('econ1').contains('Adquisición de Activos Financieros')` condition can be omitted here?

Reply:
@weilu 
Thanks for the catch! It does seem like the condition check
`~col('econ1').contains('Adquisición de Activos Financieros')` is redundant in this case.